packages/migrations/src/migrateDockerNetworkIpRange/restoreContainersToNetwork/restoreContainersToNetwork.ts

```diff
@@ -0,0 +1,31 @@
+import Dockerode from "dockerode";
+import { reconnectDisconnectedContainers } from "./reconnectDisconnectedContainers.js";
+import { recreateContainersToRecreate } from "./recreateContainersToRecreate.js";
+import { restartContainersToRestart } from "./restartContainersToRestart.js";
+
+export async function restoreContainersToNetwork({
```

Comment:
I think all the functions called inside this one should have a `catch` that makes sure that an error thrown does not prevent the others from being executed

Reply:
done!